app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :comment
```

Comment:
- [ ] Kindly remove this line , there should be no relationship between like & comment. 
- [ ] Instead of this kindly write ` belongs_to :author, class_name: 'User'` because there should be a relationship between user and likes.